src/js/components/channels/viber-channel-content.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { Component, PropTypes } from 'react';
+import { LoadingComponent } from '../../components/loading';
+import { fetchViberQRCode } from '../../services/integrations-service';
+
+
+export class ViberChannelContent extends Component {
+    static contextTypes = {
```

Comment:
can you convert this to not use `context` at all and use `connect` instead? plsplspls

I'm currently working on converting everything to `connect` and stop using `context`.